src/fastpath/fastpath.c

```diff
@@ -15,6 +15,289 @@
 #endif
 #include <benchmark/benchmark_utilisation.h>
 
+
+#ifdef CONFIG_ARCH_ARM
+static inline
+FORCE_INLINE
+#endif
+void NORETURN fastpath_vm_fault(vm_fault_type_t type)
+{
+    cap_t handler_cap;
```

Comment:
nit: this declare before use is an old style from when we compiled the kernel with c90, it's not required anymore, and actually creates extra lines of proof. So new code doesn't need to use it.

Reply:
I see, I started with a declare as used style but changed it to be more similar to the call and replyrecv fastpaths. I will change it soon.